CMake/ScmRevGen.cmake

```diff
@@ -53,9 +53,11 @@ function(configure_source_file path)
   file(REMOVE "${PROJECT_BINARY_DIR}/${path}.tmp")
 endfunction()
 
-configure_source_file("Source/Core/Common/scmrev.h")
+if(NOT INSIDE_CPACK)
```

Comment:
Maybe split this part into a separate file so you don't need the flag?

Reply:
I diidn't wanted to change too much because of possible conflicts (such as the version changing above), but it is indeed cleaner. I have done this now.